src/module/actor/sheet/vehicle.js

```diff
@@ -303,7 +299,7 @@ export class ActorSheetSFRPGVehicle extends ActorSheetSFRPG {
             const actorId = $(event.currentTarget).parents('.crew').data('actorId');
             const crew = game.actors.get(actorId);
 
-            crew.rollSkill("piloting", {event: event});
+            crew.rollSkill("piloting", {event: event}, this.actor.data.data.attributes.modifiers.piloting);
```

Comment:
See comment above, not sure if I can include this in the `options` object instead of increasing parameter count.

Reply:
Yeah, I would prefer you to directly call the DiceSFRPG.rolld20 function here instead with the properties set up through the roll formula and roll context instead.